docs/resources/Application.md

```diff
@@ -94,3 +96,7 @@
 | ----------- | ---------------- | ---------------------------------------------------------------------------------------------------------- |
 | scopes      | array of strings | the [scopes](#DOCS_TOPICS_OAUTH2/shared-resources-oauth2-scopes) to add the application to the server with |
 | permissions | string           | the [permissions](#DOCS_TOPICS_PERMISSIONS) to request for the bot role                                    |
+
+## Get Current Bot Application Information % GET /applications/@me
+
+Returns the bot's [application](#DOCS_RESOURCES_APPLICATION/application-object) object.
```

Comment:
Would add a note that the info returned from this endpoint is partial/different from the oauth2 route